slices/erlang-snmp.yaml

```diff
@@ -0,0 +1,32 @@
+package: erlang-snmp
+
+essential:
+  - erlang-snmp_copyright
+
+slices:
+  bins:
+    essential:
+      - erlang-base_bins
+      - erlang-crypto_libs
+      - erlang-mnesia_libs
+      - erlang-runtime-tools_libs
+      - erlang-snmp_config
+      - erlang-snmp_libs
+    contents:
+      /usr/bin/snmpc:
+      /usr/lib/erlang/lib/snmp-*/bin/snmpc:
```

Comment:
pls test them

Reply:
The priv files in erlang-os-mon packages are includes in the bins slice, since they are binaries not modules.